app/models/task.py

```diff
@@ -1,5 +1,25 @@
 from app import db
 
-
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True, default = None)
+    goal_id = db.Column(db.Integer, db.ForeignKey("goal.goal_id"), nullable=True)
+    def to_dict(self):
+        if Task.completed_at is True:
```

Comment:
Hm, this line says to me, "Look at the `Task` class, and find some sort of class method or class variable called `completed_at`. If that's equal to `True`, then do the following:"

For me, this is weird because I know there _isn't_ a class method or class variable called `completed_at`. I know that there's an _instance_ variable called `completed_at`, used for _instances_ of `Task`s... If I were to call an instance variable within an instance method (and `to_dict` is an instance method), then I would expect the use of `self`:

```python
if self.completed_at is True:
```